grpc-proxy/build.gradle

```diff
@@ -1,15 +1,20 @@
 import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+import io.deephaven.tools.License
 
 plugins {
   id 'com.bmuschko.docker-remote-api'
 }
 
+def dockerLicenses = License.createFrom(project).syncDockerLicense().get()
```

Comment:
mixing register and create is weird, esp for new code, should pick a convention, so we don't register (i.e. make it lazy) and then right away call get(), so it is never actually lazy.